GETTING_STARTED.md

```diff
@@ -0,0 +1,166 @@
+> This getting started guide is a work in progress. Not all sections are complete, completed sections may be a bit rough.
+
+# Getting Started with Monorepos
+
+Hi! Welcome. This guide is a very quick guide to help you set up your own monorepo, a repository that includes multiple javascript packages, all managed by a unifying root config. It aligns with our [monorepo style guide](./README.md), but is designed to walk you through the basics. If you want to have a better understanding of what decisions are being made for you, we recommend reading the guie after you've followed this tutorial.
+
+Finally, the repository we are building is based on [our monorepo starter](https://github.com/thinkmill/monorepo-starter), so if you simply want to be up and running, feel free to check that out.
+
+We will be assuming that you have [git](https://www.atlassian.com/git/tutorials/what-is-git) set up, as well as an [npm account](http://npmjs.com/). We are also using [yarn classic](https://classic.yarnpkg.com/lang/en/) for our package installs, so expect it to be installed globally. We are also assuming you are familiar with running [terminal commands](https://jes.st/the-terminal-101/)
+
+## Initialising your repository
+
+Firstly, we're going to need a repository to build our monorepo in. Running the following commands in your terminal will get you a new repository ready for work:
+
+```shell
```

Comment:
Isn't this opposite to what we are telling people to do in the monorepo-starter? Which is click "Use this template".

Reply:
This guide is how to build the starter, to teach you about monorepos. It's a getting started tutorial